merge-pr-pipeline.yml

```diff
@@ -0,0 +1,31 @@
+# Pipeline to create Github merge PRs using the dotnet bot.
+
+variables:
+# Variable group for the bot secret.
+- group: RoslynMergeBot
+- name: isDryRun
+  value: false
+- name: isAutomated
+  value: true
+
+schedules:
+- cron: "0 */12 * * *"
```

Comment:
what does this mean? 2x daily?

Reply:
Yep, it's the same cron as in https://github.com/dotnet/roslyn-tools/blob/master/src/AzureFunctionPackage/Functions/GitHubCreateMergePRs/function.json#L6 (except that azure functions use ncrontab with an additional field - https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer#ncrontab-expressions ).  

See - https://crontab.guru/every-12-hours